src/renderer/reader/components/ReaderOptions.tsx

```diff
@@ -581,55 +581,60 @@ export class ReaderOptions extends React.Component<IProps, IState> {
+                        <div className={styles.subheading}>{__("reader.settings.disposition.title")}</div>
+                        <div className={styles.center_in_tab}>
+                            <div className={styles.focus_element}>
+                                <input
+                                    id={styles.scroll_option}
+                                    type="radio"
+                                    name="disposition"
+                                    onChange={(e) => isPdf
+                                        ? this.props.pdfEventBus.dispatch("view", "scrolled")
+                                        : this.props.handleSettingChange(e, "paged", false)}
+                                    checked={isPdf
+                                        ? this.state.pdfView === "scrolled"
+                                        : !readerConfig.paged}
+                                />
+                                <label
+                                    htmlFor={styles.scroll_option}
+                                    className={isPdf
+                                        ? this.getButtonClassNamePdf(this.state.pdfView === "scrolled")
+                                        : this.getButtonClassName("paged", false)}
+                                >
+                                    <SVG svg={DefileIcon} />
+                                    {__("reader.settings.scrolled")}
+                                </label>
+                            </div>
+                            <div className={styles.focus_element}>
+                                <input
+                                    id={styles.page_option}
+                                    type="radio"
+                                    name="disposition"
+                                    onChange={(e) => isPdf
+                                        ? this.props.pdfEventBus.dispatch("view", "paginated")
+                                        : this.props.handleSettingChange(e, "paged", true)}
+                                    checked={isPdf
+                                        ? this.state.pdfView === "paginated"
+                                        : readerConfig.paged}
+                                />
+                                <label
+                                    htmlFor={styles.page_option}
+                                    className={isPdf
+                                        ? this.getButtonClassNamePdf(this.state.pdfView === "paginated")
+                                        : this.getButtonClassName("paged", true)}
+                                >
+                                    <SVG svg={PagineIcon} />
+                                    {__("reader.settings.paginated")}
+                                </label>
+                            </div>
+                        </div>
+                    </section>
+            }
```

Comment:
Post-merge suggestion: the `<section>` immediately below this one uses a different technique to test for PDF and eliminate from GUI: `hidden={this.props.isPdf}`, I think it would be better to use the same technique across the board (i.e. `hidden` attribute or `isPdf` ternary). I personally prefer the latter, as the `isPDF` state does not change in the reader view instance, so it makes total sense to completely "remove" the unused JSX fragments so that the generated DOM is as small as possible.